libs/core/src/main/java/org/elasticsearch/bootstrap/JarHell.java

```diff
@@ -277,10 +277,10 @@ private static void checkClass(Map<String, Path> clazzes, String clazz, Path jar
                         "class: " + clazz + System.lineSeparator() +
                         "exists multiple times in jar: " + jarpath + " !!!!!!!!!");
             } else {
-                throw new IllegalStateException("jar hell!" + System.lineSeparator() +
```

Comment:
You'll probably not be able to run CI with this change as we have tests that check it's working. 
It  will be faster to run `./gradlew precommit --parallel` locally it should run in a few minutes ( maybe 10+ on the first run when it also needs to download ) and will allow you to iterate quickly on some build issues